contrib/ros/README.md

```diff
@@ -0,0 +1,138 @@
+##### Customize the intera.sh script
+    $ cd ~/<YOUR_FAVORITE_WORKSPACE_NAME>
+    $ gedit intera.sh
+##### Edit the 'robot_hostname' field
+##### Edit the 'your_ip' field
+##### Verify 'ros_version' field
+update: ros_version='kinetic'
+##### Save and Close intera.sh script
+##### Initialize your SDK environment
+    $ cd ~/<YOUR_FAVORITE_WORKSPACE_NAME>
+    $ ./intera.sh
+### Setup Gazebo Simulation for Sawyer
+**Please skip this section if you don't want simulation**
+#### Installation/Prerequisites
+- Make sure you have finished [workstation setup](#setup-workstation)
+- Ensure the following software packages are installed:
+    
+    
+    $ sudo apt-get install gazebo7 ros-kinetic-qt-build ros-kinetic-gazebo-ros-control ros-kinetic-gazebo-ros-pkgs ros-kinetic-ros-control ros-kinetic-control-toolbox ros-kinetic-realtime-tools ros-kinetic-ros-controllers ros-kinetic-xacro python-wstool ros-kinetic-tf-conversions ros-kinetic-kdl-parser ros-kinetic-sns-ik-lib
+#### Sawyer Simulator Installation
+##### Install sawyer_simulator
+    $ cd ~/<YOUR_FAVORITE_WORKSPACE_NAME>/src
+    $ git clone https://github.com/RethinkRobotics/sawyer_simulator.git
+    $ wstool merge sawyer_simulator/sawyer_simulator.rosinstall
+    $ wstool update
+##### Build Source
+    $ source /opt/ros/kinetic/setup.bash
+    $ cd ~/<YOUR_FAVORITE_WORKSPACE_NAME>
+    $ catkin_make -DPYTHON_EXECUTABLE:FILEPATH=<path_to_your_rllab_env_python_interp>
+### Convert Python2 code in intera packages to Python3
+- As our environments only support Python3, we need to convert Python2 code in intera packages.
+- These can be done using rllab.contrib.ros.scripts.sawyer_2to3.sh
+##### Customize sawyer_2to3.sh
+Upgrade path:
+    
+    ros_ws=<YOUR_FAVORITE_WORKSPACE_PATH>/src/
+Remove sawyer_simulator, if you don't need it:
+
+    packages=(intera_sdk)
+##### Execute script
+## Usage
+### Start the gazebo simulation
+- This section is only for simulation user
+#### Copy the sawyer simulation launch file and simulation world file
+    $ cp <rllab.contrib.ros.envs.sawyer.sawyer_learning.launch, rllab.contrib.ros.envs.sawyer.sawyer_world_learning.launch> <YOUR_FAVORITE_WORKSPACE_PATH>/src/sawyer_simulator/sawyer_gazebo/launch
+    $ cp <rllab.contrib.ros.envs.sawyer.sawyer_learning.world> <YOUR_FAVORITE_WORKSPACE_PATH>/src/sawyer_simulator/sawyer_gazebo/worlds/
+#### Set the environment variables
+    $ cd ~/<YOUR_FAVORITE_WORKSPACE_NAME>
+    $ ./intera.sh sim
+    $ export PYTHONPATH=/opt/ros/kinetic/lib/python2.7/dist-packages:<YOUR_FAVORITE_WORKSPACE_PATH>/devel/lib/python3/dist-packages
+#### Launch the gazebo sawyer
+    $ roslaunch sawyer_gazebo sawyer_learning.launch
+### Run the training script
+#### Set the environment variables
+    $ cd ~/<YOUR_FAVORITE_WORKSPACE_NAME>
+    $ ./intera.sh sim
+    $ source activate rllab3
+#### Run the training script
+    $ python <rllab.contrib.ros.envs.example_launchers>/<launcher>
```

Comment:
Ex. python rllab/contrib/ros/envs/example_launchers/trpo_gazebo_sawyer_pnp.py

Reply:
What I added:
#### Run the training script
**Make sure that $RLLAB is in your $PYTHONPATH** \
Ex.

    $ python $RLLAB/contrib/ros/envs/example_launchers/trpo_gazebo_sawyer_pnp.py